### PR TITLE
New package: linux4.20-4.20.0

### PR DESCRIPTION
**Pull request by Hoshpak**
[skip ci]

### CODE REVIEW

File: srcpkgs/linux4.20/files/x86_64-dotconfig [hunk 1]
```diff
@@ -0,0 +1,8952 @@
...
+CONFIG_THREAD_INFO_IN_TASK=y
+
+#
+# General setup
+#
+CONFIG_INIT_ENV_ARG_LIMIT=32
+# CONFIG_COMPILE_TEST is not set
+CONFIG_LOCALVERSION="_1"
+# CONFIG_LOCALVERSION_AUTO is not set
+CONFIG_BUILD_SALT="n"
```
Comment (philipgg) [inline srcpkgs/linux4.20/files/x86_64-dotconfig]: Why `CONFIG_BUILD_SALT="n"` ? I do not think that makes sense.

File: srcpkgs/linux4.20/template [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+ /usr/lib/modules/${_kernver}/modules.dep
+ /usr/lib/modules/${_kernver}/modules.dep.bin
+ /usr/lib/modules/${_kernver}/modules.symbols
+ /usr/lib/modules/${_kernver}/modules.symbols.bin
+ /usr/lib/modules/${_kernver}/modules.alias
+ /usr/lib/modules/${_kernver}/modules.alias.bin
+ /usr/lib/modules/${_kernver}/modules.devname"
+
+do_configure() {
+	# 4.19 misses Documentation/DocBook. We ship the directory from 4.12 here.
```
Comment (philipgg) [inline srcpkgs/linux4.20/template]: it's 4.20

File: srcpkgs/linux4.20/files/x86_64-dotconfig (hunk 1 above)
Comment (philipgg) [reply srcpkgs/linux4.20/files/x86_64-dotconfig]: ( idem for 4.19: https://github.com/void-linux/void-packages/blob/master/srcpkgs/linux4.19/files/x86_64-dotconfig#L23 )

Comment (Hoshpak) [reply srcpkgs/linux4.20/files/x86_64-dotconfig]: carried over from 4.19. Indeed the default is to set it to an empty string, I changed it. Thanks for catching that.

**Pull request merged**